text/chapter11.md

```diff
@@ -623,20 +625,21 @@ class (Applicative f, Plus f) <= Alternative f
 
 `Alternative` provides two new combinators: the `empty` value, which provides a prototype for a failing computation, and the `alt` function (and its alias, `<|>`) which provides the ability to fall back to an _alternative_ computation in the case of an error.
 
-The `Data.List` module provides two useful functions for working with type constructors in the `Alternative` type class:
+The `Data.Array` module provides two useful functions for working with type constructors in the `Alternative` type class:
```

Comment:
Noting that this issue was fixed early, although should have used `Array` instead of `List`.
https://github.com/paf31/purescript-book/issues/130